2020-05/robot-name/robot-name.js

```diff
@@ -2,46 +2,121 @@
 // provided as a convenience to get your started writing code faster.
 
 export class Robot {
-  constructor(name){
-    this.nameHistory = []
-    this.name = this.randomName()
+  // A list of all robot names used.
+  static nameHistory = {}
+
+  // A private robot name.
+  #name = ''
```

Comment:
Since the test for whether or not this property can be externally modified is passing, I'm assuming this works like magic :sparkles: :wink: - This uses the new babel dependency you added?